src/test/java/seedu/address/logic/commands/DeleteCommandTest.java

```diff
@@ -47,28 +46,24 @@ public void execute_validNameFilteredList_success() {
         showPersonWithName(model, ALICE.getName());
 
         Person personToDelete;
-        try {
-            personToDelete = model.findByName(ALICE.getName());
-            DeleteCommand deleteCommand = new DeleteCommand(ALICE.getName());
+        personToDelete = model.findByName(ALICE.getName());
+        DeleteCommand deleteCommand = new DeleteCommand(ALICE.getName());
 
-            String expectedMessage = String.format(DeleteMessages.MESSAGE_DELETE_PERSON_SUCCESS,
-                    DeleteMessages.format(personToDelete));
+        String expectedMessage = String.format(DeleteMessages.MESSAGE_DELETE_PERSON_SUCCESS,
+                DeleteMessages.format(personToDelete));
 
-            Model expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
-            expectedModel.deletePerson(personToDelete);
-            showNoPerson(expectedModel);
+        Model expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+        expectedModel.deletePerson(personToDelete);
+        showNoPerson(expectedModel);
 
-            assertCommandSuccess(deleteCommand, model, expectedMessage, expectedModel);
-        } catch (CommandException e) {
-            e.printStackTrace();
-        }
+        assertCommandSuccess(deleteCommand, model, expectedMessage, expectedModel);
     }
 
     @Test
     public void execute_invalidNameFilteredList_throwsCommandException() {
         showPersonWithName(model, ALICE.getName());
 
-        Name invalidName = BENSON.getName();
+        Name invalidName = new Name("blbeo");
```

Comment:
It might be good idea to maintain a constant with INVALID_NAME so that it can be reused in most of the test cases

Reply:
agreed!